feature-detects/clipboard/clipboard.js

```diff
@@ -0,0 +1,74 @@
+/*!
+{
+  "name": "Clipboard API",
+  "property": "clipboard",
+  "tags": ["clipboard"],
+  "authors": ["Markel Ferro (@MarkelFe)"],
+  "async": true,
+  "warnings": ["These tests currently require document.body to be present"],
+  "notes": [{
+    "name": "MDN Docs Clipboard Object",
+    "href": "https://developer.mozilla.org/en-US/docs/Web/API/Clipboard"
+  }, {
+    "name": "MDN Docs Clipboard API",
+    "href": "https://developer.mozilla.org/en-US/docs/Web/API/Clipboard_API"
+  }]
+}
+!*/
+/* DOC
+It tests for the whole clipboard API. For sub-features check other tests
+*/
+define(['Modernizr', 'addTest'], function(Modernizr, addTest) {
+  Modernizr.addAsyncTest(function() {
+    var result;
+    // var name;
+    // var props = ['read', 'readText', 'write', 'writeText'];
+    if (navigator.clipboard) {
+      addTest('clipboard', true);
+      /*
+      // The sub-modules checked only if API is available to avoid Edge crashes
+      for (var i = 0; i < props.length; i++) {
+        if (navigator.clipboard[props[i]]) {
+            result = true;
+        } else {
+            result = false;
+        }
+        name = props[i].toLowerCase();
+        Modernizr.addTest('clipboard.' + name, result);
+      }
```

Comment:
If someone could help me with this part would appreciate it. As far as my debugging goes the if statement checks correctly whether the browser supports the feature, the problem seems to come more during the process of adding it to the array.

Reply:
Btw, to test it you'll need to uncomment lines 24 and 25 and probably comment the work around

Reply:
Okay, I've found the error 😅, i just had to remove Modernizr from line 37 🤦‍♂️. Submitting a PR soon.